k8s/aat/common/ia/case-api.yaml

```
@@ -2,7 +2,7 @@
 apiVersion: flux.weave.works/v1beta1
 kind: HelmRelease
 metadata:
-  name: ia-case-api
+  name: case-api
```

Please prefix with your product name, they have to be unique across all apps, case-api is a name that could clash 
```suggestion
  name: ia-case-api
```